prime-angular-frontend/src/app/modules/enrolment/pages/self-declaration/self-declaration.component.html

```diff
@@ -12,10 +12,28 @@
     <app-page-subheader>
       <ng-container appPageSubheaderTitle>Self-Declaration</ng-container>
       <ng-container appPageSubheaderSummary>
-        Self-declaration information is collected under the authority of the Information Management Regulation of the
-        Pharmaceutical Services Act and is protected under the Freedom of Information and Protection of Privacy Act.
-        Self-declaration information is used to verify the applicant is in good standing with their respective colleges
-        and to determine eligibility for and access to PharmaNet.
+        In these Self Declaration Questions, “unlawful or improper action” means any of the following actions taken
+        <u>by you</u> or a <u>by a person for which you were responsible:</u>
```

Comment:
With this typo correction necessary, https://bcgovmoh.atlassian.net/browse/PRIME-2301?focusedCommentId=14734, is there a way to refactor the three areas in code where this Intro exists into a common component so any future changes are synchronized?